draft-rfcs/working-group-terminology.md

```diff
@@ -32,57 +34,57 @@ To address this confusion this RFC proposes breaking up the single term, into th
 
 [reference-level-explanation]: #reference-level-explanation
 
+### Common Aspects of Working Groups and Project Groups
+
+Before going into the differences between these groups it's important to start with what it is shared between them.
+
+- Both groups have a charter that defines the scope and intent of the group.
+- Both groups have a GitHub repository hosted under the rust-lang organization (or separate organisation if necessary) containing the charter and instructions for how community members can monitor the group's progress and/or participate.
+- Groups have at least one shepherd who acts as the leader of the group and is responsible for writing the initial charter, handling adminstrative and communication tasks, as well as delegating responsibilities to other members in the group.
+  - [This blog post][shep-post] explains in detail the role of the shepherds.
```

Comment:
I think it's "ok" to cite my blog post for now, and I've done it a few times too, but I'd prefer to introduce an RFC or some official document. I don't generally consider my blog posts to be normative or representative of anything but my random thoughts and opinions.

Reply:
I'm also not 100% sure if shepherds are a universal concept -- I think they apply more to project groups. For example, I think the embedded domain WG has its own notion of teams which have shepherds (@jamesmunns can clarify).

Reply:
(Maybe this a terminology point we should address, though -- i.e., the shepherd role is basically a "lead", and maybe we just want to be reserving lead for teams in general?)